src/utils/crypto.js

```diff
@@ -50,44 +34,40 @@ function hmac (key, message) {
   return h.digest()
 }
 
-function hmacJsonForPskCondition (obj, sharedSecret) {
+function hmacPacketForPskCondition (packet, sharedSecret) {
```

Comment:
To make this even more clear what it's for I think we should rename this to `generateFulfillment`.